src/styles/blocks/_stars.scss

```diff
@@ -13,10 +13,9 @@
     background-repeat: no-repeat;
     background-position: 50%;
   }
-
 }
 
-.stars--4 :nth-of-type(-n + 4) {
+.stars--4 :nth-child(-n+4) {
```

Comment:
add all modifiers `.stars--1, .stars--2...`

Reply:
Hi! Added changes and deployed project. Take a look, please